sql/ddl_internal.sql

```diff
@@ -565,3 +565,11 @@ BEGIN
     END LOOP;
 END
 $BODY$;
+
+
+--documentation of these function located in chunk_index.h
+CREATE OR REPLACE FUNCTION _timescaledb_internal.chunk_index_recreate_create(chunk_index_oid OID) RETURNS OID
```

Comment:
Although these functions are utilities for addons, we should still be able to have tests for them. If we have no tests that exercise these functions we don't know if some other change will break them.

Reply:
tests added